package.json

```diff
@@ -5,7 +5,7 @@
   ],
   "name": "sendgrid-rest",
   "description": "HTTP REST client, simplified for Node.js.",
-  "version": "2.4.0",
+  "version": "2.4.1",
```

Comment:
Please set this back to 2.4.0. I will be updating the version number at the next release. Thanks!

Reply:
Got it, will commit the corresponding package-lock.json too